src/Layer.js

```diff
@@ -111,7 +111,7 @@ export default class Layer extends Observable {
   getVisibleChildren() {
     for (let i = 0; i < this.children.length; i += 1) {
       if (this.children[i].getVisible()) {
-        return true;
+        return this.children[i];
```

Comment:
it should return an array with all the visible children shouldn't it?

Reply:
Now it does :)

Reply:
I mean, now ;)